lib/datadog/security/waf.rb

```diff
@@ -0,0 +1,68 @@
+require 'libsqreen'
+require 'datadog/security/assets'
+
+module Datadog
+  module Security
+    module WAF
+      class Args < Hash; end
+
+      class Rules < Hash; end
+
+      module_function
+
+      # TODO: logger
+      def logger
+        @logger ||= ::Logger.new(STDOUT)
+        @logger.level = ::Logger::DEBUG
+        #@logger.level = ::Logger::WARN
+        @logger.debug { 'waf logger enabled' }
+        @logger
+      end
+
+      def rules
+        @rules ||= JSON.dump(JSON.parse(Assets.waf_rules))
```

Comment:
I think this class would benefit from being an instance object, which initializes `@rules`, `@name` and `@logger` at construction time.

Reply:
Indeed. I'm waiting on some changes on the C++ WAF API to update this further.

Reply:
Minor: It also perhaps makes sense to have `Asserts` parse the JSON (or the new format directly), rather than keeping two copies of stuff in memory -- an unparsed JSON string, and then the parsed format.